<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgInspect</class>
 <widget class="QDialog" name="dlgInspect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>244</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inspect</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblCountdown">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>Trebuchet MS</family>
        </font>
       </property>
       <property name="text">
        <string>REBOOTING.. WAIT {x} SECONDS...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlActions">
       <item>
        <widget class="QPushButton" name="btnGood">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Passed</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnBad">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>Trebuchet MS</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Failed</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
